src/components/app/MenuButtons/MetaInfo.tsx

```diff
@@ -242,34 +245,64 @@ class MetaInfoPanelImpl extends React.PureComponent<Props, State> {
               {formatTimestamp(meta.profilingEndTime - meta.profilingStartTime)}
             </div>
           ) : null}
-          {meta.profilingStartTime === undefined && meta.startTime ? (
-            <div className="metaInfoRow">
-              <span className="metaInfoLabel">
-                <Localized id="MenuButtons--metaInfo--main-process-started">
-                  Main process started:
-                </Localized>
-              </span>
-              {_formatDate(meta.startTime)}
-            </div>
-          ) : null}
-          {meta.endTime ? (
-            <div className="metaInfoRow">
-              <span className="metaInfoLabel">
-                <Localized id="MenuButtons--metaInfo--main-process-ended">
-                  Main process ended:
-                </Localized>
-              </span>
-              {_formatDate(meta.endTime)}
-            </div>
-          ) : null}
+          {this.props.timelineUnit === 'bytes' ? (
+            // For size profiles, show the file name and size instead of timestamps
+            <>
+              {meta.fileName ? (
+                <div className="metaInfoRow">
+                  <span className="metaInfoLabel">
+                    <Localized id="MenuButtons--metaInfo--file-name">
+                      File name:
+                    </Localized>
+                  </span>
+                  {meta.fileName}
+                </div>
+              ) : null}
+              {meta.endTime && meta.startTime !== undefined ? (
+                <div className="metaInfoRow">
+                  <span className="metaInfoLabel">
+                    <Localized id="MenuButtons--metaInfo--file-size">
+                      File size:
+                    </Localized>
+                  </span>
+                  {formatBytes(meta.endTime - meta.startTime)}
```

Comment:
I think I'd prefer a separate fileSize property over this. Yes it's somewhat consistent but it's also quite strange to read.